src/allotropy/parsers/thermo_fisher_nanodrop_eight/nanodrop_eight_reader.py

```diff
@@ -21,6 +21,6 @@ def read(cls, contents: IOType) -> pd.DataFrame:
             # Prevent pandas from rounding decimal values, at the cost of some speed.
             float_precision="round_trip",
         )
-        raw_data = raw_data.rename(columns=lambda x: x.strip())
+        raw_data = raw_data.rename(columns=lambda x: x.strip().lower())
```

Comment:
This refactor and all the changes associated with it are making it harder to see the real fixes. Could you please either undo this change or split it into a separate PR with this depending on it?

Reply:
This is necessary for the fix.  Part of the reason concentration was not being captured in test files 1 and 2  was due to inconsistency in header capitalization.  Does it help if the column header capitalization changes and the additional logic for handling the concentration are separate commits?

Reply:
Yep, totally fine if the column header capitalization changes are a separate PR!